internal/filewatcher/term_illumos.go

```diff
@@ -0,0 +1,4 @@
+package filewatcher
+
+const tcGet = 0x540d
+const tcSet = 0x540e
```

Comment:
You are almost certainly right that these constants likely won't change ever, but I wonder if you can't use `unix.TCGETS` and `unix.TCSETS`, respectively? If nothing else, having a name makes it a bit less mysterious from where these values originate.

These constants do seem to be present in `golang.org/x/sys/unix` when `GOOS=illumos`, at least with recent versions.

```console
$ GOOS=illumos go doc golang.org/x/sys/unix.TCGETS
...
	TCGETS                        = 0x540d
...
```